egs2/TEMPLATE/vc1/setup.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+help_message=$(cat << EOF
+Usage: $0 <target-dir>
+EOF
+)
+
+
+if [ $# -ne 1 ]; then
+    log "${help_message}"
+    log "Error: 1 positional argument is required."
+    exit 2
+fi
+
+
+dir=$1
+mkdir -p "${dir}"
+
+if [ ! -d "${dir}"/../../TEMPLATE ]; then
+    log "Error: ${dir}/../../TEMPLATE should exist. You may specify wrong directory."
+    exit 1
+fi
+
+targets=""
+
+# Copy
+for f in README.md conf local; do
+    target="${dir}"/../../TEMPLATE/vc1/"${f}"
+    cp -r "${target}" "${dir}"
+    targets+="${dir}/${target} "
+done
+
+
+# Symlinks to TEMPLATE/vc1
+target=../../TEMPLATE/vc1/vc.sh
+ln -sf "${target}" "${dir}"
+targets+="${dir}/${target} "
+
+
+# Symlinks to TEMPLATE/tts1
+target=../../TEMPLATE/tts1/path.sh
```

Comment:
since you already has a templdate in vc, how about symlink to there (I understand it's exactly the same for now, but just in case)